spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/ProviderRoleMapping.java

```diff
@@ -56,6 +57,14 @@ public ProviderRoleMapping(boolean mapOauthScopes, Map<String, String> roleMappi
 		this.roleMappings = roleMappings;
 	}
 
+	public boolean isParsePathParts() {
```

Comment:
Do you mind adding some javadocs here for this new property? 

Unfortunately there is no README in this repo to update w/ the relevant info. The only place I see this currently documented is indirectly in [Spring Cloud Dataflow](https://github.com/spring-cloud/spring-cloud-dataflow/blob/main/spring-cloud-dataflow-docs/src/main/asciidoc/configuration-local.adoc#role-mappings). I am not suggesting we update that though.

Reply:
Added javadoc